code/modules/atmospherics/machinery/components/fusion/hfr_main_processes.dm

```diff
@@ -548,8 +548,10 @@
 
 	if(moderator_list[/datum/gas/oxygen] > 150)
 		if(iron_content > 0)
-			iron_content = max(iron_content - 0.5, 0)
-			moderator_internal.gases[/datum/gas/oxygen] -= 10
+			var/max_iron_removable = 0.00415
+			var/iron_removed = max(max_iron_removable * delta_time, iron_content)
```

Comment:
don't you mean min() ? if you leave max wouldn't it remove all the iron content since it will always be higher

Reply:
Good catch, last minute refactor strikes again